internal/services/cdn/cdn_frontdoor_origin_group_resource.go

```diff
@@ -188,7 +188,7 @@ func resourceCdnFrontDoorOriginGroupRead(d *pluginsdk.ResourceData, meta interfa
 	ctx, cancel := timeouts.ForRead(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
-	id, err := parse.FrontDoorOriginGroupID(d.Id())
+	id, err := parse.FrontDoorOriginGroupIDInsensitively(d.Id())
```

Comment:
this is coming from the Resource ID and so should be being parsed case-sensitively:

```suggestion
	id, err := parse.FrontDoorOriginGroupID(d.Id())
```